### ISSUE TITLE
AUR mokutil-git cannot install

### ISSUE THREAD

**Issue by AzAgarampur** [linux-surface/linux-surface#244]
Hey there,
This is probably a dumb question but I can't get this to work. I'm installing Arch linux on SL3 (intel). I am trying to install the secure boot signing key. As the install guide said, it needs a dependency from AUR. So I've installed `yay` and I'm trying to install the dependency `mokutil-git` by simply running `yay -S mokutil-git`. Everything goes fine until I get this error
```./configure: line 4612: syntax error near unexpected token `OPENSSL,`
./configure line 4612: 'PKG_CHECK_MODULES(OPENSSL, openssl >= 0.9.8)'```

I have tried to edit the `configure` file by adding `[]` around the `OPENSSL, openssl >= ...` but it seems to overwrite my changes every time I run `yay`. Is there a way to get this to work?

**Comment by qzed** [comment]
I can't reproduce this, is your system up-to-date?

**Comment by AzAgarampur** [comment]
Yep. I just installed arch with the standard install instructions from the archlinux wiki. I haven't installed any of the surface kernel/package stuff yet.

**Comment by qzed** [comment]
You did run `pacman -Syu` after the installation? What is your package version for `mokutil-git` (the updated version, which gets shown during build via `==> Updated version: mokutil-git <version>`)? What is your autoconf version (`autoreconf --version`)?

**Comment by AzAgarampur** [comment]
Yup, I did right after the install. pacman reports all packages are up to date. There is nothing to do.

autoreconf is version 2.69
updated version reports mokutil-git 0.4.0.r7.g6bfb714-1

**Comment by qzed** [comment]
Okay, weird. Those are the exact same versions I have and work for me. Have you tried removing the cached files yet (`rm -rf ~/.cache/yay/mokutil-git/`)? Does `yay -S mokutil` work? If so we could find a workaround.

**Comment by AzAgarampur** [comment]
Yeah I removed the cache from `yay` and tried again. That didn't work. So I tried removing the entire cache as well. No luck once again. Let me try running `yay -S mokutil`.

**Comment by AzAgarampur** [comment]
Nope, the exact same thing happens when I run yay -S mokutil

**Comment by qzed** [comment]
Since we both have the same versions installed I think this might be a dependency issue. Unfortunately I can't test that with a clean Arch container at the moment since the LXC image server seems to be down...

Do you have `pkgconf` and/or `base-devel` installed?

**Comment by AzAgarampur** [comment]
Woah. I have both of those packages installed but I decided to reinstall them anyways. My 'base-devel' actually showed an increase in the upgrade size. Then I ran 'yay -S mokutil-git' and it worked. I guess something wasn't fully installed from 'base-devel'. I was successful able to install the 'linux-surface-secureboot-mok' package now.

Thanks!

**Comment by qzed** [comment]
Hmm, odd. Well, I'm glad it works now.
